src/NzbDrone.Core/Indexers/Definitions/Aither.cs

```diff
@@ -0,0 +1,57 @@
+using System.Collections.Generic;
+using NLog;
+using NzbDrone.Common.Http;
+using NzbDrone.Core.Configuration;
+using NzbDrone.Core.Indexers.Definitions.UNIT3D;
+using NzbDrone.Core.Messaging.Events;
+
+namespace NzbDrone.Core.Indexers.Definitions
+{
+    public class Aither : Unit3dBase
+    {
+        public override string Name => "Aither";
+        public override string BaseUrl => "https://aither.cc/";
+        public override IndexerPrivacy Privacy => IndexerPrivacy.Private;
```

Comment:
Can we get description copied over from yml?

Reply:
done